.circleci/config.yml

```diff
@@ -97,6 +97,8 @@ commands:
             conda create -n hydra python=<< parameters.py_version >> pywin32 virtualenv==20.0.33 -qy
             conda activate hydra
             pip install nox dataclasses --progress-bar off
+            pip uninstall pyzmq
+            pip install pyzmq
```

Comment:
should definitely not be solved as a part of this PR.

Reply:
:) never meant to, this was not intended